.gitignore

```diff
@@ -7,4 +7,5 @@ node_modules
 .vscode
 
 /data
+/app/static/fonts
```

Comment:
Perhaps it would be good to mention how to acquire/set up the font files if you don’t have this directory yet? At least I’m a bit lost now about how to get the required fonts. :D